cypress/e2e/dataset.cy.ts

```diff
@@ -6,8 +6,9 @@ describe('Dataset details page', () => {
       .first()
       .then($cardName => {
         // Navigate to the first dataset details page.
-        cy.getBySel('dataset-card').first().click();
-        cy.location('pathname').should('include', '/dataset');
+        cy.getBySel('dataset-card-name').first().click();
+        // Wait for the page to load
```

Comment:
Minor: this comment doesn't end with a `.`, but e.g. line 8 and 12 do. Use one style, for consistency?